src/services/completions.ts

```diff
@@ -1505,7 +1506,8 @@ namespace ts.Completions {
                     // TODO(jfreeman): Account for computed property name

```

Comment:
I've removed Jason's name, but I think computed property names might still not be handled specially - if a computed property is returned from `getNameOfDeclaration`, `getEscapedTextOfIdentifierOrLiteral` will quietly return `undefined`.